docs/user-tutorial/benchmarks/micro-benchmarks.md

```diff
@@ -16,7 +16,7 @@ which is defined as the time range from the beginning of the launch API call to
 #### Metrics
```

Comment:
I think our goal for this PR is to giev a consistent metric name prefixes, e.g. “time”, “bandwidth” -- so it’s easy to understand whether deviation is good or bad.